test/unit/forms/ConsignmentReferencesSpec.scala

```diff
@@ -35,62 +39,64 @@ class ConsignmentReferencesSpec extends BaseSpec {
   "Consignment References mapping" should {
     "return errors for empty fields" in {
       val inputData = ConsignmentReferences("", "")
-      val errors = ConsignmentReferences.form().fillAndValidate(inputData).errors
+      val errors = ConsignmentReferences.form(arrive).fillAndValidate(inputData).errors
```

Comment:
Instead of using either `arrive` or `depart` you could put these values into a collection and then run tests for each element of this collection. Similar amount of code, but test coverage is much better.

Reply:
I did consider this but as the only affect of using one or the other value is that a different I18n message key is selected (rather than an actual change in the code's behaviour) it did warrant the effort.